src/FeedsCleanup/packages.config

```diff
@@ -0,0 +1,6 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Microsoft.Web.Xdt" version="2.1.0" targetFramework="net452" />
+  <package id="NDesk.Options" version="0.2.1" targetFramework="net452" />
```

Comment:
Well. I generally not high on the idea of using the framework currently building to build the tool to build the next build. Better to keep it old but simply rather than scratch our head when a build is busted then tools are broken therefore we can't build the next build to fix the broken build. 

And well if you insist, I can do it in next PR.